languages.yaml

```diff
@@ -1311,6 +1311,7 @@ Jupyter:
       - "#9E9E9E"
     chip: "#DA5B0B"
   serialization: jupyter-notebooks
+  line_types: [code, comments]
```

Comment:
Actually, we may not need it for `Jupyter`. Seems like tokei treats jupyter notebooks only as a holder for other languages with 0 as a count for all line types:

![image](https://user-images.githubusercontent.com/13710835/213878539-7cf8af5b-3864-4e04-9fcd-bbc20d760603.png)